sdk/ai/ai-projects/src/agents/utils.ts

```diff
@@ -28,3 +28,31 @@ export function fromFunctionDefinition(functionDefintion: FunctionDefinition): F
         function: functionDefintion
     }
 }
+
+/** Types of connection tools sued to configure an agent */
+export enum connectionToolType {
+  /** Bing grounding search tool */
+  BingGrounding = "bing_grounding",
+  /** Microsoft Fabric tool */
+  MicrosoftFabric = "microsoft_fabric",
+  /** Sharepoint tool */
+  SharePointGrounding = "sharepoint_grounding",
+  /** Azure AI search tool */
```

Comment:
Does AI search tool support connections ?

Reply:
Mistakenly included that, removing now - thanks for catching that!